modern-business/style-editor.css

```diff
@@ -807,7 +816,10 @@ body .wp-block[data-align="full"] {
 }
 
 @media only screen and (min-width: 600px) {
-  body .wp-block[data-align="full"] {
+  
+  body .wp-block[data-align="full"],
+  body .wp-block[data-type="a8c/template"] .wp-block[data-align="full"],
```

Comment:
Instead of using these data attribute selectors, is it possible for us to set a class on these blocks like we do for any other block?

Using the data attribute selectors will be fragile because this could change at any point in a future Gutenberg update.

Reply:
Great question! I did attempt this, but I wasn't able to get it working. The problem is that the theme css targets the width of every `.wp-block`, so we needed to somehow override that for the post content block. The custom class name (`.post-content-block`) we have set for this block is much further down the DOM tree, so adjusting it doesn't really help us override the size of the actual block:
<img width="1131" alt="Screen Shot 2019-07-17 at 2 46 32 PM" src="https://user-images.githubusercontent.com/6265975/61414005-bc11e080-a8a1-11e9-9957-11cf47211e27.png">

I'm not sure how we would otherwise target that top level block div. I'm very open to suggestions; as a beginner, there may be some Gutenberg way to do this I'm not aware of, or even a more clever css selector!

Reply:
Blocks generate classes with the following pattern: `wp-block-${namespace}-${block}-${name}`. These are usually generated for us, but in the case of dynamic blocks we need to generate that ourselves since we fully control the render callback.

So for a more stable selector we can do something like:
`body.block-editor-page .wp-block-a8c-post-content`

After updating the dynamic block render callbacks to include this.

Reply:
Hm, I'm not sure if this targets what I'd like to target.

For example, with a paragraph block, I would want to target the outer layer which is selected. But it looks like the autogenerated classname is much further down the dom tree:
<img width="1298" alt="Screen Shot 2019-07-17 at 4 02 16 PM" src="https://user-images.githubusercontent.com/6265975/61417583-61ca4d00-a8ac-11e9-9ce0-f08ee3ae7aa1.png">

Maybe I'm missing something?

Reply:
`.wp-block[data-type]` seems ok to me. Since we need to modify the outer layer of the block, I don't think we can use any other selector.

Reply:
Summarizing a slack chat with @noahtallen and @vindl and @gwwar  I think we'll +1 @mmtr's suggestion and stick with this.

If we're afraid of behavior changing, we can add a follow up E2E test to see if this still exists before upgrading Gutenberg, and we can add a note to PCYsg-mfI-p2 for followup.